website/docs/docs/cloud/canvas-interface.md

```diff
@@ -35,29 +36,28 @@ The operator toolbar above the canvas contains the different transformation oper
 <Lightbox src="/img/docs/dbt-cloud/canvas/operators.png" width="90%" title="Use the operator toolbar to perform different transformation operations." />
 
 Here the following operators are available:
-
 #### Input
 
 Input operators configure source data:
-- **Input model**: Select the model and columns you want to use.
+- **Model explorer**: Select the model and columns you want to use. 
```

Comment:
it looks like you need to select and drag the operators?
```suggestion
- **Model explorer**: Select and drag the model and columns you want to use into the Canvas.
```

Reply:
ok understood - its just a little confusing because i tried selecting it and it does nothing but wiggle. semantics maybe and dont feel strongly but i was confused when i was read this